src/utils/navigation.ts

```diff
@@ -386,4 +386,9 @@ export const NAVIGATION: Navigation = {
       config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL,
     ],
   },
+  [config.UPDATE_MANAGE_TRUSTS_REVIEW_THE_TRUST_URL]: {
+    currentPage: config.UPDATE_MANAGE_TRUSTS_REVIEW_THE_TRUST_PAGE,
+    previousPage: () => config.UPDATE_MANAGE_TRUSTS_INTERRUPT_URL,
+    nextPage: [config.UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_URL]
+  },
```

Comment:
I'm pretty sure you don't need this config in here and can remove it - as the POST trusts method you're using handles the validation/errorList stuff itself and not through the generic checkValidations middleware.